### DISCUSSION TITLE
Difficulty icons' tooltip shows an incorrect star rating in some difficulties

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#15944] [Q&A] [answered]
There's this weird issue in some difficulties of different mapsets in my carousel.
The tooltip on their difficulty icon shows a wrong (old?) star rating. The correct one is visible in the info wedge.
(Unfortunately, I don't remember if there was a change in the star rating after the date I downloaded these maps.)

Re-importing the maps seem to solve the issue, but I guess there must be some way to keep this value up-to-date.

Before re-importing:
![before](https://user-images.githubusercontent.com/43073074/144755569-36a36b98-2506-4be1-ad10-980782ff1949.png)

After re-importing:
![after](https://user-images.githubusercontent.com/43073074/144755609-98c9c016-c6e9-409e-8810-683bba03f3f7.png)

**Comment by bdach** [comment]
This is known. The icons on the carousel don't update since https://github.com/ppy/osu/pull/10497 since it was causing performance drops. This will be fixed at some point when star difficulties are stored to DB and periodically updated accordingly, which should remove most of the overhead.

**Comment by frenzibyte** [comment, marked as answer]
There's already an issue thread tracking this (https://github.com/ppy/osu/issues/10242).
